base_tier_validation/models/tier_validation.py

```diff
@@ -857,9 +857,14 @@ def get_view(self, view_id=None, view_type="form", **options):
                 node.append(new_node)
                 _merge_view_fields(all_models, new_models)
             excepted_fields = self._get_all_validation_exceptions()
+            readonly_fields = self.env['ir.model.fields'].search([('model', '=', self._name), ('readonly', '=', True)]).mapped('name')
             for node in doc.xpath("//field[@name][not(ancestor::field)]"):
                 if node.attrib.get("name") in excepted_fields:
                     continue
+                if node.attrib.get("name") in readonly_fields and not node.attrib.get("readonly"):
+                    """Readonly field with readonly not set on form -> odoo will set this as readonly=True
+                    Make sure not to override"""
```

Comment:
```suggestion
                    # Readonly field with the attribute readonly not set on form -> will become editable
```
This should be a comment not a string